validator/client/payload_attestation.go

```diff
@@ -83,3 +87,25 @@ func (v *validator) signPayloadAttestation(ctx context.Context, p *ethpb.Payload
 	// Marshal the signature into bytes
 	return sig.Marshal(), nil
 }
+
+// waitUntilPtcDuty waits until PTC duty time which is defined as
+func (v *validator) waitUntilPtcDuty(ctx context.Context, slot primitives.Slot) {
+	ctx, span := trace.StartSpan(ctx, "validator.waitToSlotTwoThirds")
+	defer span.End()
+
+	startTime := slots.StartTime(v.genesisTime, slot)
+	dutyTime := startTime.Add(3 * slots.DivideSlotBy(4))
+	wait := prysmTime.Until(dutyTime)
+	if wait <= 0 {
```

Comment:
I don't think this can ever happen as above things are constant and positive always

Reply:
It can be negative given genesis time and input slot. TestWaitUntilPtcDuty_Return for example. Should get -40s here